pubspec.yaml

```
@@ -41,7 +41,7 @@ dev_dependencies:
   analyzer: ^6.8.0
   archive: ^3.1.2
   crypto: ^3.0.0
-  dart_style: ^2.0.0
+  dart_style: ">=2.0.0 <4.0.0"
```

Perhaps it's better to update this to `^3.0.0` to block 2.x, especially when code change is required for updating, such that the updated code may no longer be compatible with 2.x.
```suggestion
  dart_style: ^3.0.0
```